repobee_sanitizer/_sanitize_file.py

```diff
@@ -24,16 +24,27 @@ def _sanitize_file(
         """A callback function that runs the sanitization protocol on a given
         file.
 
-        Args:
-            args: Parsed and processed args from the RepoBee CLI.
-            api: A platform API instance.
-        Returns:
-            A mapping (str -> List[plug.Result]) that RepoBee's CLI can use for
-            output.
+        Args: args: Parsed and processed args from the RepoBee CLI. api: A
+            platform API instance.
+
+        Returns: An optional plug.PlugResult if the syntax is invalid,
+            otherwise nothing.
```

Comment:
```suggestion
        Args:
            args: Parsed and processed args from the RepoBee CLI.
            api: A platform API instance.
        Returns:
            An optional plug.Result if the syntax is invalid, otherwise nothing.
```